packages/shared/src/components/profile/ProfileSettingsMenu.tsx

```diff
@@ -28,10 +28,25 @@ import { anchorDefaultRel } from '../../lib/strings';
 import type { NavItemProps } from '../drawers/NavDrawerItem';
 import { LogoutReason } from '../../lib/user';
 import { useAuthContext } from '../../contexts/AuthContext';
+import { usePrompt } from '../../hooks/usePrompt';
+import { ButtonColor } from '../buttons/Button';
 
 const useMenuItems = (): NavItemProps[] => {
   const { logout } = useAuthContext();
   const { openModal } = useLazyModal();
+  const { showPrompt } = usePrompt();
+  const onLogout = useCallback(async () => {
+    const shouldLogout = await showPrompt({
+      title: 'Are you sure?',
+      className: { buttons: 'mt-5' },
+      okButton: { title: 'Logout', color: ButtonColor.Ketchup },
+      cancelButton: { title: 'Stay' },
```

Comment:
Please check with tomer, we recently removed all "other named cancel buttons" to just say cancel.
Was actually at the point where I wanted to remove the cancel override back then 😅

So pretty sure it should just be Cancel

Reply:
"Cancel" has been the default. Reversed the placement now.

![Screenshot 2024-04-05 at 11 25 53 PM](https://github.com/dailydotdev/apps/assets/13744167/fec82dc9-04f6-4095-90e4-e39c11210865)